packages/core/src/components/Image.story.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Image from './Image';
+import lunar from ':storybook/images/lunar-logo.png';
+
+storiesOf('Core/Image', module)
+  .addParameters({
+    inspectComponents: [Image],
```

Comment:
maybe if you add `happo: { delay: 250 },` as a param, it'll help them generate as the images load?

Reply:
Hmmm doesn't seem to have changed anything

Reply:
I'm not really sure why they aren't loading. Happo *should* wait for them: https://github.com/happo/happo.io#image-loading